research/src/sources/semantic.json5

```diff
@@ -236,7 +236,68 @@
     },
     {
       "name": "Image",
-      "url": "https://semantic-ui.com/elements/image.html"
+      "url": "https://semantic-ui.com/elements/image.html",
+      "definition": "An image is a graphic representation of something",
+      "concepts": [
+        {
+          "name": "default",
+          "description": "Unless size is specified, image will use its original dimensions filling the container",
+          "image": "semantic-image-default.png"
+        },
+        {
+          "name": "fluid",
+          "openUIName": "fit",
+          "description": "An image can take up the width of its container",
+          "image": "semantic-image-fluid.png"
```

Comment:
This image includes a frame around the image and an HTML snippet. Let's trim it down to just an image of the component.